src/pages/CategoryList.tsx

```diff
@@ -0,0 +1,150 @@
+import { Button, Icon } from '@ahaui/react';
+import PaginationTable from 'components/common/table/PaginationTable';
+import useAppSelector from 'hooks/useAppSelector';
+import useTypedDispatch from 'hooks/useTypedDispatch';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
```

Comment:
Please re-order import library to top.
Import order:
- Library
- Internal component